<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QFrame" name="centralframe">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <widget class="QMenu" name="menuSessions">
     <property name="title">
      <string>Sess&amp;ions</string>
     </property>
    </widget>
    <addaction name="actionAddDataset"/>
    <addaction name="separator"/>
    <addaction name="menuSessions"/>
    <addaction name="actionSessionManager"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menu_View">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <widget class="QMenu" name="menu_Layout">
     <property name="title">
      <string>View &amp;Layout</string>
     </property>
     <addaction name="actionLayoutSingleView"/>
     <addaction name="actionLayoutDualView"/>
     <addaction name="actionLayoutDualViewVertical"/>
     <addaction name="actionLayoutOneLargeTwoSmallViews"/>
     <addaction name="actionLayoutOneLargeThreeSmallViews"/>
     <addaction name="actionLayoutQuadView"/>
    </widget>
    <addaction name="actionSceneManagement"/>
    <addaction name="separator"/>
    <addaction name="actionWaypoints"/>
    <addaction name="actionBoundingBoxes"/>
    <addaction name="separator"/>
    <addaction name="actionResizeWindow"/>
    <addaction name="actionFullScreen"/>
    <addaction name="menu_Layout"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionOnlineManual"/>
    <addaction name="separator"/>
    <addaction name="actionReportABug"/>
    <addaction name="separator"/>
    <addaction name="actionAboutQt"/>
    <addaction name="actionAboutMet3D"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menu_View"/>
   <addaction name="menu_Help"/>
  </widget>
  <action name="actionExit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionLayoutSingleView">
   <property name="text">
    <string>&amp;1 Single View</string>
   </property>
   <property name="shortcut">
    <string>Alt+1</string>
   </property>
  </action>
  <action name="actionLayoutDualView">
   <property name="text">
    <string>&amp;2 Dual View</string>
   </property>
   <property name="shortcut">
    <string>Alt+2</string>
   </property>
  </action>
  <action name="actionLayoutOneLargeTwoSmallViews">
   <property name="text">
    <string>&amp;4 One Large, Two Small Views</string>
   </property>
   <property name="shortcut">
    <string>Alt+4</string>
   </property>
  </action>
  <action name="actionLayoutQuadView">
   <property name="text">
    <string>&amp;6 Quad View</string>
   </property>
   <property name="shortcut">
    <string>Alt+6</string>
   </property>
  </action>
  <action name="actionLayoutOneLargeThreeSmallViews">
   <property name="text">
    <string>&amp;5 One Large, Three Small Views</string>
   </property>
   <property name="shortcut">
    <string>Alt+5</string>
   </property>
  </action>
  <action name="actionSceneManagement">
   <property name="text">
    <string>&amp;Scene Management</string>
   </property>
   <property name="shortcut">
    <string>F4</string>
   </property>
  </action>
  <action name="actionLayoutDualViewVertical">
   <property name="text">
    <string>&amp;3 Dual View, Vertical</string>
   </property>
   <property name="shortcut">
    <string>Alt+3</string>
   </property>
  </action>
  <action name="actionWaypoints">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show &amp;Waypoints</string>
   </property>
   <property name="shortcut">
    <string>F2</string>
   </property>
  </action>
  <action name="actionFullScreen">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Full Screen</string>
   </property>
   <property name="shortcut">
    <string>F11</string>
   </property>
  </action>
  <action name="actionAddDataset">
   <property name="text">
    <string>&amp;Add Dataset</string>
   </property>
   <property name="shortcut">
    <string comment="F3"/>
   </property>
  </action>
  <action name="actionOnlineManual">
   <property name="text">
    <string>&amp;Online Manual</string>
   </property>
   <property name="toolTip">
    <string>Open the online manual in a web browser</string>
   </property>
  </action>
  <action name="actionReportABug">
   <property name="text">
    <string>&amp;Report a Bug</string>
   </property>
   <property name="toolTip">
    <string>Report a bug in the online issue tracker</string>
   </property>
  </action>
  <action name="actionAboutMet3D">
   <property name="text">
    <string>About &amp;Met.3D</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="text">
    <string>&amp;About Qt</string>
   </property>
  </action>
  <action name="actionResizeWindow">
   <property name="text">
    <string>&amp;Resize Window</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionBoundingBoxes">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show &amp;Bounding Boxes</string>
   </property>
   <property name="shortcut">
    <string>F3</string>
   </property>
  </action>
  <action name="actionSessionManager">
   <property name="text">
    <string>&amp;Session Manager...</string>
   </property>
   <property name="shortcut">
    <string>F6</string>
   </property>
  </action>
  <action name="actionSessions">
   <property name="text">
    <string>Sessions</string>
   </property>
  </action>
  <action name="actionDefault">
   <property name="text">
    <string>default</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>activated()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
